website/blog/2025-10-28-dynamo-on-aks/index.md

```diff
@@ -0,0 +1,291 @@
+    shifting GPUs from decode or spinning up additional ones.
+    * At night, when email generation jobs dominate, Dynamo reallocates GPUs
+    back to decode to optimize throughput.
+    * When load drops, resources scale back down.
+
+    **Result**: SLOs are met consistently without over or under provisioning,
+    controlling costs while maintaining performance.
+
+1. **Recomputing shared context is wasteful**
+
+    Many requests within the same session reuse the same product or user
+    context but unnecessarily recompute the KV cache each time, wasting
+    valuable GPU resources that could be spent serving other user requests.
+
+    **Dynamo Solution**: LLM-aware routing maintains a map of KV cache across
+    large GPU clusters and directs requests to the GPUs that already hold the
+    relevant KV cache, avoiding redundant computation.
+
+    In our e-commerce example:
+
+    * A user browses five similar items in one session.
+    * Dynamo routes all requests to the same GPU that already has the user’s
+    or product’s context cached.
+
+    **Result**: Faster response times, lower latency, reduced GPU usage.
+
+1. **KV cache growth blows past GPU memory**
+
+    With many concurrent sessions and large input sequence lengths, the
+    KV cache (product data + user history) can exceed available GPU memory.
+    This can trigger evictions, leading to costly re-computations or inference
+    errors.
+
+    **Dynamo Solution**: The KV Block Manager (KVBM) offloads cold/unused KV
+    cache data to CPU RAM, NVMe, or networked storage freeing valuable GPU
+    memory for active requests.
+
+    In our e-commerce example:
+
+    * Without cache offloading: increasing number of concurrent sessions
+    per GPU increases latency due to KV cache evictions and recomputations
+    * With Dynamo: GPUs can support higher concurrencies while maintaining
+    low latency
+
+    **Result**: Higher concurrency at lower cost, without degrading user
+    experience.
+
+## Enterprise-scale inference experiments:  Dynamo with GB200, running on AKS
+
+We set out to deploy the popular open-source
+[GPT-OSS 120B](https://huggingface.co/openai/gpt-oss-120b) reasoning model
+using Dynamo on AKS on GB200 NVL72, adapting the
+[SemiAnalysis InferenceMAX](https://inferencemax.semianalysis.com/) recipe
+for a large scale, production-grade environment. Our approach: leverage Dynamo
+as the inference server and swap GB200 NVL72 nodes in place of B200, scaling
+the deployment across multiple nodes.
+
+Our goal was to replicate the performance results reported by SemiAnalysis,
+but at a larger scale within an AKS environment, proving that enterprise-scale
+inference with cutting-edge hardware and open-source models is not only
+possible, but practical.
+
+### AKS Deployment Overview
+
+Ready to build the same setup? Our comprehensive guide walks you through
+each stage of the deployment:
+
+1. *Setting up your foundation.* Configure GPU node pools and prepare your

```

Comment:
The link says to skip GPU driver install and install the NVIDIA GPU Operator. Is that a prerequisite? Would it work without that?
Also what are the plans to publish something like this in our docs?